src/diagnostics/LogPointsDebugAdapter.ts

```diff
@@ -4,10 +4,12 @@ import {
     Thread
 } from 'vscode-debugadapter';
 import { DebugProtocol } from 'vscode-debugprotocol';
+import * as WebSiteModels from '../../node_modules/azure-arm-website/lib/models';
```

Comment:
this import is weird.  why are you referencing something from inside node_modules? Can you just say `import * as WebSiteModesl from 'azure-arm-website'`?

Reply:
There are two d.ts files in the referenced package, models and operations. Another option would be (since you are using just the type User):
```
import { User } from 'azure-arm-website/lib/models';
```

Reply:
Used the way @whiskeyjay suggested.